build-rpi-raspbian.sh

```diff
@@ -69,6 +69,7 @@ cp $sourcefolder/first-boot.sh $buildfolder/boot/first-boot.sh
 touch $buildfolder/boot/first-boot.log
 cp $sourcefolder/openhabian.raspbian.conf $buildfolder/boot/openhabian.conf
 unix2dos $buildfolder/boot/openhabian.conf
+cp $sourcefolder/webif.sh $buildfolder/boot/webif.sh
```

Comment:
Please add to the pine script as well.

Reply:
is there python also installed on pine?

Reply:
This will be taken care of in https://github.com/openhab/openhabian/pull/358 will finish it up later this month.